<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="edit_music">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_music">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择音乐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_music">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_current_media" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);</string>
    </property>
    <widget class="QLabel" name="label_current_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>381</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_artist_tip">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>歌手:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_title_tip">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>标题:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_artist">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_title">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>55</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="slider_current_music_position">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_music_pos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_music_length">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_album_tip">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>专辑:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_album">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_load_music">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>741</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_load_output_music">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>122</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>→ 加载 ↑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_output_tip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>输出路径:</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_custom_info" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>731</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(222, 222, 222);</string>
     </property>
     <widget class="QPushButton" name="btn_select_picture">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_picture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>加入图片</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_picture">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_album">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_artist">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_album_tip">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>专辑:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title_tip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>标题:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_artist_tip">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>歌手:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_combine">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>合成</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_output_music_path">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>122</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_src_tip">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Github 源码&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/BensonLaur/BesMP3Editor.git&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#666666;&quot;&gt;BesMP3Editor &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_readme">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;操作说明&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://www.cnblogs.com/BensonLaur/p/11075125.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#666666;&quot;&gt;cnblog &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>edit_music</zorder>
   <zorder>label_music</zorder>
   <zorder>btn_select_music</zorder>
   <zorder>widget_current_media</zorder>
   <zorder>btn_load_music</zorder>
   <zorder>btn_play</zorder>
   <zorder>btn_pause</zorder>
   <zorder>btn_stop</zorder>
   <zorder>label_src_tip</zorder>
   <zorder>label_readme</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
